parquet/src/bloom_filter/mod.rs

```diff
@@ -16,7 +16,61 @@
 // under the License.
 
 //! Bloom filter implementation specific to Parquet, as described
-//! in the [spec](https://github.com/apache/parquet-format/blob/master/BloomFilter.md).
+//! in the [spec][parquet-bf-spec].
+//!
+//! # Bloom Filter Size
+//!
+//! Parquet uses the [Split Block Bloom Filter][sbbf-paper] (SBBF) as its bloom filter
+//! implementation. For each column upon which bloom filters are enabled, there will be an SBBF
+//! stored in the metadata for each row group in the parquet file. The size of each filter is
```

Comment:
I think technically the bloom filter is not stored in the metadata, but rather the metadata stores the offset and length of the bloom filter location ([length](https://docs.rs/parquet/latest/parquet/file/metadata/struct.ColumnChunkMetaData.html#method.bloom_filter_length) and [offset](https://docs.rs/parquet/latest/parquet/file/metadata/struct.ColumnChunkMetaData.html#method.bloom_filter_offset)) -- this is important as it means you can read the metadata without the bloom filter information

Perhaps we can change this to something like

```suggestion
//! implementation. For each column upon which bloom filters are enabled, the offset and length of an SBBF
//! is stored in  the metadata for each row group in the parquet file. The size of each filter is
```